packages/cli/src/bulk.ts

```diff
@@ -28,9 +28,13 @@ bulkExportCommand
     'optional target directory to save files from the bulk export operations.'
   )
   .action(async (options) => {
-    const { exportLevel, types, since, targetDirectory } = options;
-    const medplum = await createMedplumClient(options);
+    if (!checkForProfile(options)) {
```

Comment:
Can this `checkForProfile` call be done inside `createMedplumClient`?  That way we don't need to duplicate it on every command.  `createMedplumClient` can throw if the options are invalid.